src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -97,6 +97,10 @@ public OtlpExporterOptions()
     /// * http://localhost:4317 for <see cref="OtlpExportProtocol.Grpc"/>
     /// * http://localhost:4318 for <see cref="OtlpExportProtocol.HttpProtobuf"/>.
     /// </summary>
+    /// <remarks>
+    /// When using <see cref="OtlpExportProtocol.HttpProtobuf"/>, the full URL MUST be provided, including the signal-specific path v1/{signal}.
```

Comment:
Not a blocker for this PR, what if the user doesn't specify a full URL or doesn't include signal-specific path - do we validate the input and give the caller an exception?

Reply:
We do not validate the input. The Uri is used as is. The only way they would know is via sdk internal logs when the request fails. Exception would be a better option, will look in to it.

Reply:
The [documentation comments](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5385/files#diff-1e7c29e7cbb12b62aa8515d38a5bb21af38f73d4594559d323bc180d49090fefR98) still mention the default value for `OtlpExportProtocol.HttpProtobuf` to be `http://localhost:4318`. Could we update that to the actual default values that would be used by the exporter? That is, `http://localhost:4318/v1/traces` for traces and so on?

Reply:
This was resolved in #5434